pandas/core/sparse/frame.py

```diff
@@ -846,6 +846,13 @@ def applymap(self, func):
         return self.apply(lambda x: lmap(func, x))
 
 
+def _nan_to_np_nan(value):
+    """Normalize nan values to singleton np.NaN object so that when NaNs are
```

Comment:
can you point to the test that is failing. I don't want to address this in this PR. This is non-trivial and needs to be common code. ok with xfailing those tests (and making an issue)